pkg/apiserver/metrics/metrics.go

```diff
@@ -48,6 +48,8 @@ var (
 		prometheus.SummaryOpts{
 			Name: "apiserver_request_latencies_summary",
 			Help: "Response latency summary in microseconds for each verb and resource.",
+			// Make the sliding window of 1h.
+			MaxAge: time.Hour,
```

Comment:
There is no requirement for these to be the same. Although it's probably more intuitive to have the equal in all places.